test.rb

```diff
@@ -1,2 +1,13 @@
 puts 'Hello World'
 puts "hollow world"
+puts "More Badly formatted" +  "Code to flag"  + "rubocop Warnings"
+def test_func (test_input)
```

Comment:
W: Unused method argument - `test_input`. If it's necessary, use `_` or `_test_input` as an argument name to indicate that it won't be used. You can also write as `test_func(*)` if you want the method to accept any arguments but don't care about them.